app/controllers/partners/authorized_family_members_controller.rb

```diff
@@ -0,0 +1,61 @@
+module Partners
+  class AuthorizedFamilyMembersController < BaseController
```

Comment:
As for specs for there, I generally do like the request/controller specs but maybe we can use `magic_test` to make integration tests and that would be enough. Let's talk about that in our sync today.